_commands/tools/sftp.md

```diff
@@ -3,22 +3,152 @@
 ~~~ bash
-*short* example of command usage and output
+$ sftp user@hostname
+Password: *********
+Connected to hostname.
+sftp> put filename.txt
+sftp> mput file1.txt file2.txt file3.txt
+sftp> get file.txt
+sftp> mget file1.txt file2.txt file3.txt
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
 
-#### Example command
+#### `sftp user@hostname`
+~~~ bash
+$ sftp mmdarden@login.engin.umich.edu
+Password: **********
+Connected to login.engin.umich.edu.
+sftp>
+~~~
+
+##### Break it down
+
+ * This is how you begin the sftp process. It is done the same way as when you SSH into a machine, except you use the sftp command instead of the ssh command. You must have access to the machine and provide a password. You can try this by remote accessing CAEN with your uniqname@login.engin.umich.edu and providing your kerberos password as the password.
+
+#### `sftp> ls`
+~~~ bash
+sftp> ls
+remoteDirectory1
+remoteFile1.txt
+sftp> lls
+localDirectory1
+localFile1.txt
+~~~
+
+
+##### Break it down
+
+ * Use the `ls` command to print the contents of the current directory on the remote machine. Use the `lls` command to print the contents of the current directory on your local machine.
+
+
+ #### `sftp> cd directory`
+~~~ bash
+sftp> cd remoteDirectory1
+sftp> lcd localDirectory1
+~~~
+
+##### Break it down
+
+ * Uset the `cd` command to change your current directory on the remote machine. Use the `lcd` command to change your current directory on your local machine.
+
+ #### `sftp> put filename`
+~~~ bash
+sftp> put localFile1.txt
+~~~
+
+##### Break it down
+
+ * Upload a single file from your local machine to the remote machine.
+
+ #### `sftp> mput filename1 filename2 filename2`
+~~~ bash
+sftp> mput localFile1.txt localFile2.txt localFile3.txt
+~~~
+
+##### Break it down
+
+ * Upload several files from your loacl machine to the remote machine.
+
+ #### `sftp> put -r directory/`
+~~~ bash
+sftp> mkdir localDirectory1
+sftp> put -r localDirectory/
+~~~
 
 ##### Break it down
 
-#### Example command
+ * To upload a directory from your local machine to the remote machine, first make a new destination directory on the remote machine, and then upload the directory from your local machine using `put`.
+
+ #### `sftp> get filename`
+~~~ bash
+sftp> get remoteFile1.txt
+~~~
+
+##### Break it down
+
+ * Download a single file from the remote machine to your local machine.
+
+ #### `sftp> mget filename1 filename2 filename2`
+~~~ bash
+sftp> mget remoteFile1.txt remoteFile2.txt remoteFile3.txt
+~~~
 
 ##### Break it down
+
+ * Download several files from the remote machine to your local machine.
+
+ #### `sftp> get -r directory`
+~~~ bash
+sftp> get -r remoteDirectory1
+~~~
+
+##### Break it down
+
+ * Download a directory and its contents from the remote machine to your local machine.
+
+ #### `sftp> ?`
+~~~ bash
```

Comment:
Rather than having this as a code block, could you put this into a table? It would make it much more readable.